modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/OclLoader.java

```diff
@@ -17,10 +19,15 @@
 @Component
 public class OclLoader {
 
-    private final ObjectMapper mapper = new ObjectMapper();
+    private final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
do we really need two ObjectMappers here?

Reply:
yes, This PR is only for the API refactor. I will get rid of the old codes soon.